{
	"schema_version": "1.6.0",
	"id": "PYSEC-2023-270",
	"modified": "2024-11-21T14:57:15.140612Z",
	"published": "2023-03-06T23:15:00Z",
	"aliases": [
		"CVE-2022-4134",
		"GHSA-5gp5-vxj6-4257"
	],
	"details": "A flaw was found in openstack-glance. This issue could allow a remote, authenticated attacker to tamper with images, compromising the integrity of virtual machines created using these modified images.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:N/I:L/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "glance",
				"purl": "pkg:pypi/glance"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						}
					]
				}
			],
			"versions": [
				"15.0.2",
				"17.0.1",
				"18.0.0",
				"18.0.0.0b1",
				"18.0.0.0rc1",
				"18.0.1",
				"19.0.0",
				"19.0.0.0b1",
				"19.0.0.0rc1",
				"19.0.0.0rc2",
				"19.0.1",
				"19.0.2",
				"19.0.3",
				"19.0.4",
				"20.0.0",
				"20.0.0.0b1",
				"20.0.0.0b2",
				"20.0.0.0b3",
				"20.0.0.0rc1",
				"20.0.0.0rc2",
				"20.0.1",
				"20.1.0",
				"20.2.0",
				"21.0.0",
				"21.0.0.0b1",
				"21.0.0.0b2",
				"21.0.0.0rc1",
				"21.0.0.0rc2",
				"21.1.0",
				"22.0.0",
				"22.0.0.0b2",
				"22.0.0.0b3",
				"22.0.0.0rc1",
				"22.1.0",
				"22.1.1",
				"23.0.0",
				"23.0.0.0b2",
				"23.0.0.0b3",
				"23.0.0.0rc1",
				"23.0.0.0rc2",
				"23.1.0",
				"24.0.0",
				"24.0.0.0rc1",
				"24.1.0",
				"24.2.0",
				"24.2.1",
				"25.0.0",
				"25.0.0.0b2",
				"25.0.0.0b3",
				"25.0.0.0rc1",
				"25.1.0",
				"26.0.0",
				"26.0.0.0b2",
				"26.0.0.0b3",
				"26.0.0.0rc1",
				"26.1.0",
				"27.0.0",
				"27.0.0.0b1",
				"27.0.0.0b2",
				"27.0.0.0rc1",
				"27.1.0",
				"28.0.0",
				"28.0.0.0b2",
				"28.0.0.0rc1",
				"28.0.1",
				"28.1.0",
				"29.0.0",
				"29.0.0.0b1",
				"29.0.0.0b2",
				"29.0.0.0b3",
				"29.0.0.0rc1"
			],
			"database_specific": {
				"source": "https://github.com/pypa/advisory-database/blob/main/vulns/glance/PYSEC-2023-270.yaml"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://wiki.openstack.org/wiki/OSSN/OSSN-0090"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2147462"
		},
		{
			"type": "REPORT",
			"url": "https://bugs.launchpad.net/glance/+bug/1990157"
		}
	]
}
